storage/bucket.go

```diff
@@ -666,9 +673,8 @@ type LifecycleRule struct {
 type LifecycleAction struct {
 	// Type is the type of action to take on matching objects.
 	//
-	// Acceptable values are "Delete" to delete matching objects and
-	// "SetStorageClass" to set the storage class defined in StorageClass on
-	// matching objects.
+	// Acceptable values are "Delete", "SetStorageClass", and
```

Comment:
Would that have the potential for confusing users as the type is string (for example, they may try passing in the string `"DeleteAction"` rather than `"Delete"` or `DeleteAction`)?

Reply:
I would call it `storage.DeleteAction` because that's how it would be used in a setting where the library is imported.